shared/common-adapters/clickable-box.js.flow

```diff
@@ -2,10 +2,12 @@
 import React, {Component} from 'react'
 
 export type Props = {
+  children?: any,
   style?: Object,
-  onClick: () => void,
-  hoverColor?: ?string,
-  backgroundColor?: ?string,
+  onClick?: ?() => void,
```

Comment:
hover / background are desktop only and aren't really explicitly defined. we just pass it through to the flatbutton and don't use it at all internally